ui/components/homepage/new-policy-view.js

```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { Link } from '../../routes';
+import Link from '../link';
```

Comment:
Will we need to remove the `<a>` from all of the `Link`s?

Reply:
Yup, pushing an update with this